src/main/java/org/igniterealtime/smack/inttest/xep0133/AbstractAdHocCommandIntegrationTest.java

```diff
@@ -52,10 +52,14 @@ public class AbstractAdHocCommandIntegrationTest extends AbstractSmackIntegratio
     public final AbstractXMPPConnection adminConnection;
     SmackIntegrationTestEnvironment environment;
 
-    public AbstractAdHocCommandIntegrationTest(SmackIntegrationTestEnvironment environment) throws SmackException, IOException, XMPPException, InterruptedException, InvocationTargetException, InstantiationException, IllegalAccessException {
+    public AbstractAdHocCommandIntegrationTest(SmackIntegrationTestEnvironment environment) throws SmackException, IOException, XMPPException, InterruptedException, InvocationTargetException, InstantiationException, IllegalAccessException, TestNotPossibleException
+    {
         super(environment);
         this.environment = environment;
 
+        if (sinttestConfiguration.adminAccountUsername == null) {
+            throw new TestNotPossibleException("This test requires an admin account to be configured.");
```

Comment:
How do you feel about pointing them at more docs? 

```suggestion
            throw new TestNotPossibleException("This test requires an admin account to be configured. Configuration instructions available at https://xmpp-interop-testing.github.io/");
```